crates/flux-fixpoint/src/lib.rs

```diff
@@ -233,15 +232,7 @@ impl<T: Types> fmt::Display for Task<T> {
 
 impl<T: Types> fmt::Display for KVar<T> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(
-            f,
-            "(var ${} ({})) // {}",
-            self.kvid,
-            self.sorts
-                .iter()
-                .format_with(" ", |sort, f| f(&format_args!("({sort})"))),
-            self.comment
-        )
+        write!(f, "(var ${} ({})) // {}", self.kvid, self.sorts.iter().join(" "), self.comment)
```

Comment:
Unrelated to this PR, but how hard would be to change the parser to accept `;` for comments instead of `//`?

Reply:
You "nerdsniped" me into doing it; will just put it in this PR.

It was a boring but tedious thing to add in the [SMTParser](https://github.com/ucsd-progsys/liquid-fixpoint/pull/700)

Reply:
That was the intention :P